src/nuget/Microsoft.DotNet.BuildTools.nuspec

```diff
@@ -19,13 +19,15 @@
     </dependencies>
   </metadata>
   <files>
-    <file src="Microsoft.DotNet.Build.Tasks\package\**\*.*" target="lib" />
+    <file src="Microsoft.DotNet.Build.CloudTestTasks\Microsoft.ServiceBus.dll" target="lib" />
```

Comment:
Should we consider putting these assemblies in a subfolder under lib given they are netfx specific and might conflict with the other .NET Core versions in the root?